nifi-nar-bundles/nifi-site-to-site-reporting-bundle/nifi-site-to-site-reporting-task/src/main/java/org/apache/nifi/reporting/AbstractSiteToSiteReportingTask.java

```diff
@@ -140,8 +164,16 @@
             .sensitive(true)
             .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
             .build();
+    static final PropertyDescriptor RECORD_WRITER = new PropertyDescriptor.Builder()
+            .name("record-writer")
+            .displayName("Record Writer")
+            .description("Specifies the Controller Service to use for writing out the records.")
```

Comment:
The description here should mention that it is only used when the reporting task is configured to use a Record Writer. For the one you added, it has two modes, but future reporting tasks need not offer an alternative.

Reply:
Left it as-is since I added the property Record Writer only in the task reporting task for now.